coralproject/talk
# bulk operations on User Detail comments

## What does this PR do?

adds the ability to change the status of comments in bulk from the User Detail panel

## How do I test this PR?

- go to the moderation queue and click the username on a comment to open the UserDetail panel
- tick a few checkboxes
- the UI should show the # of selected comments
- Accept/Reject the comments.
